kvbc/include/v4blockchain/detail/latest_keys.h

```diff
@@ -115,6 +115,23 @@ class LatestKeys {
 
   ::rocksdb::CompactionFilter* getCompFilter() { return &comp_filter_; }
 
+  std::optional<categorization::Value> getValue(BlockId latest_block_id,
+                                                const std::string& category_id,
+                                                const std::string& latest_version,
+                                                const std::string& key) const;
+  void multiGetValue(BlockId latest_block_id,
+                     const std::string& category_id,
+                     const std::string& latest_version,
+                     const std::vector<std::string>& keys,
+                     std::vector<std::optional<categorization::Value>>& values) const;
+  std::optional<categorization::TaggedVersion> getNearestLatestVersion(const std::string& category_id,
+                                                                       const std::string& latest_version,
+                                                                       const std::string& key) const;
+  void multiGetNearestLatestVersion(const std::string& category_id,
```

Comment:
same

Reply:
Thanks @cloudnoize , i changed the name to multiGetLatestVersion